openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -232,98 +243,87 @@
   implementation_status: not applicable
   narrative:
     - text: |
-        Organizational analysis and correlation of audit records across
-        different repositories to gain organization-wide
-        situational awareness is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
This suggested text does not appear to relate to the NIST control. Reference https://nvd.nist.gov/800-53/Rev4/control/Au-6#enhancement-3:


AU-6(3) | AUDIT REVIEW, ANALYSIS, AND REPORTING \| CORRELATE AUDIT REPOSITORIESThe organization analyzes and correlates audit records across different repositories to gain organization-wide situational awareness.
-- | --

Reply:
We marked this not applicable but explained that OpenShift can send the data to 3rd party systems that can correlate data.  i.e. 3rd party software and policies will be responsible for this control.

Reply:
@curt-matthews Agree on the intent. Could the IdP language be removed, since that does not address the control language?